src/main/java/io/swagger/petstore/model/Category.java

```diff
@@ -15,7 +15,7 @@ public class Category {
   private String name;
 
   @XmlElement(name = "id")
-  @Schema(example = "1")
+  @Schema(examples = "1")
```

Comment:
`examples` field is list of example values. This seems to be just redefining field name without taking into account it's type. 

This comment applies to other code changes redefining `examples` as well.